pkg/velero/server/config.go

```diff
@@ -75,7 +75,7 @@ type ServerConfig struct {
 	// How often (in nanoseconds) 'maintain' is run for backup repositories by default.
 	// +optional
 	RepoMaintenanceFrequency *time.Duration `json:"default-repo-maintain-frequency,omitempty"`
-	// How long (in nanoseconds) to wait by default before backups can be garbage collected. (default is 720 hours)
+	// How often (in nanoseconds) garbage collection checks for expired backups. (default once per hour)
```

Comment:
I would change "default once per hour" to "default is 1 hour" for consistency. I believe all other places uses this syntax

Reply:
Sure, I will patch it.

Reply:
Done.